tests/cpp/operator/CMakeLists.txt

```diff
@@ -3,28 +3,7 @@
 # See LICENSE for license information.
 
 add_executable(test_operator
-               test_cast.cu
-               test_cast_current_scaling.cu
-               test_cast_dbias.cu
-               test_cast_dbias_dgelu.cu
-               test_cast_gated_swiglu.cu
-               test_cast_mxfp8_gated_swiglu.cu
-               test_qdq.cu
-               test_cast_mxfp8.cu
-               test_dequantize_mxfp8.cu
-               test_transpose.cu
-               test_cast_transpose.cu
-               test_cast_transpose_current_scaling.cu
-               test_cast_transpose_dbias.cu
                test_cast_transpose_dbias_dgelu.cu
-               test_cast_transpose_dgeglu.cu
-               test_act.cu
-               test_normalization.cu
-               test_normalization_mxfp8.cu
-               test_multi_cast_transpose.cu
-               test_multi_padding.cu
-               test_causal_softmax.cu
-               test_swizzle.cu
```

Comment:
Those tests should not be removed, right?

Reply:
This is for debug